### PR TITLE
Removing AWS Credentials to test EC2 role

### PR DESCRIPTION
**Pull request by sbair-va**
Jenkles is now using the EC2 role 'project-devtools-role'. The AWS credentials have been commented out for all users on the instance and removed from the Jenkinsfile for the QA Deployer. Testing deployments.

### CODE REVIEW

**Comment by sbair-va** [comment]
The following apps have been deployed successfully without the Jenkins-provided AWS credentials:

Exemplar
Carma
Carma-fms-connector
Community Care
Data Query
Hotline
Mock-EE
Monitoring
Squares
Urgent Care

**Comment by bschofield-va** [comment]
`DU_AWS_ACCESS_KEY_ID` and `DU_AWS_SECRET_ACCESS_KEY` are ultimately passed to the applications running in Kubernetes and used to pulled configuration files from s3.

You said the applications deployed, but can you confirm that they are configured correctly? Do the worker nodes have a security role that allows them s3 access?

Deployment units for all products are going to need to be updated to remove usages of `DU_AWS_ACCESS_KEY_ID` and `DU_AWS_SECRET_ACCESS_KEY`, and https://github.com/department-of-veterans-affairs/health-apis-deployer/blob/qa/deployment-unit.md needs to be updated.

File: Dockerfile [hunk 1]
```diff
@@ -45,4 +45,4 @@ RUN curl -fsSL https://get.docker.com | sh
 #
 # Odd... this yum command succeeds down here at the bottom, but fails at the top.
 #
-RUN yum install -yqq openssl
\ No newline at end of file
+RUN yum install -yqq openssl
```
Comment (bschofield-va) [inline Dockerfile]: I've seen `yum install -yqq` in several places as it's generically copied around. `-yqq` is supported by `apt-get` but not `yum`.

```
$  yum install -yqq less
Ignored option -q, -v, -d or -e (probably due to merging: -yq != -y -q)
... install anyway ...
```

Comment (sbair-va) [reply Dockerfile]: Changed to yum install -y -q

**Comment by sbair-va** [comment]
@bschofield-va All apps have the DU_AWS_ACCESS_KEY_ID and DU_AWS_SECRET_ACCESS_KEY variables removed and have been deployed successfully with this branch.

**Comment by sbair-va** [comment]
Modified ReadMe

**Comment by sbair-va** [comment]
@bschofield-va dq fails to deploy using the qa branch, but succeeds using this branch (when the credentials aren't passed with blank values).

https://tools.health.dev-developer.va.gov/jenkins/job/department-of-veterans-affairs/job/health-apis-deployer/job/task%252FDEVOPS-229-switch-credentials-to-ec2-role/57/console

**Pull request merged**